utils/gyb.py

```diff
@@ -1242,16 +1242,20 @@ def succ(a):
             ast = parse_template(args.file, f.read())
     if args.dump:
         print(ast)
+
     # Allow the template to open files and import .py files relative to its own
     # directory
+    saved_cwd = os.getcwd()
     os.chdir(os.path.dirname(os.path.abspath(args.file)))
     sys.path = ['.'] + sys.path
```

Comment:
Could we use the computed path and prepend that to sys.path instead of of doing the `chdir`?

Reply:
Possibly.  That might even be a better solution; amendments welcome.